shell/server/grain-server.js

```diff
@@ -557,8 +551,8 @@ Meteor.methods({
 
       SandstormEmail.send({
         to: emailAddress,
-        envelopeFrom: envelopeFrom,
```

Comment:
Might be nice to be retain the explicitness about what `envelopeFrom` we intend to use here, as the next pass of changes(picking a different From: address) would cause that to break if we didn't add the envelopeFrom back in.

Ditto the other `SandstormEmail.send` callsites.